OpenEdXMobile/src/main/java/org/edx/mobile/http/cache/CacheManager.java

```diff
@@ -12,71 +12,99 @@
 import org.edx.mobile.util.Sha1Util;
 
 import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileOutputStream;
 import java.io.IOException;
 import java.nio.charset.Charset;
 
+/**
+ * The cache manager for HTTP responses. The cache is stored on the filesystem, within the
+ * application directory.
+ *
+ * @deprecated This is deprecated since with the transition to OkHttp, we're now relying on it's own
+ * automatic caching mechanism. However, this is still being kept around as a read-only resource for
+ * a while, being queried as a last resort by as a last resort by the CustomCacheQueryInterceptor,
+ * to facilitate the transition without any user-facing issues. This may be removed later after a
+ * significant percentage of the user base have upgraded to a version that uses the OkHttp API in
+ * the Courseware module.
```

Comment:
In this case, are you referring to the user as the other classes that consume this code?